app/assets/stylesheets/mobile/markdown_editor.scss

```diff
@@ -0,0 +1,22 @@
+.md-editor {
+  border: 1px solid $light-grey;
+  border-radius: $btn-border-radius-base;
+  overflow: hidden;
```

Comment:
`overflow: hidden` is already added to the textarea by autosize and this line here looks unnecessary.